experimental/packages/web-common/src/DefaultIdGenerator.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SessionIdGenerator } from './types/SessionIdGenerator';
+
+export class DefaultIdGenerator implements SessionIdGenerator {
```

Comment:
same question here - if users can use composition instead, I'd advocate against exporting the class to make future changes easier. 🙂